script.js

```diff
@@ -0,0 +1,129 @@
+const tttBox = document.querySelectorAll(".ttt_square")
+const playerOneScore = document.querySelector(".p1score")
+const playerTwoScore = document.querySelector(".p2score")
+const winnerText = document.querySelector(".winner_text")
+const playerOneArrow = document.querySelector(".arrowp1")
+const playerTwoArrow = document.querySelector(".arrowp2")
+const playerOnePiece = "X"
+const playerTwoPiece = "O"
+let playerOneTurn = true;
+let playerTwoTurn = false;
+
+document.querySelector(".reset_button").addEventListener('click', () => {
+    playerOneTurn = true
+    clearBoard()
+})
+
+document.querySelector(".reset_score").addEventListener('click', () => {
+    playerOneScore.textContent = 0
+    playerTwoScore.textContent = 0
+})
+
+Array.prototype.forEach.call(tttBox, box => { 
+    box.addEventListener('click', () => {
+        if(playerOneTurn && box.textContent === "") {
+            box.textContent = playerOnePiece
+        } else if(playerTwoTurn && box.textContent === "") {
+            box.textContent = playerTwoPiece
```

Comment:
here you'd be able to do else if (!playerOneTurn or playerOneTurn === false) etc